packages/data-wizard/src/analyzer/index.ts

```diff
@@ -14,6 +14,8 @@ export type TypeSpecifics = 'null' | 'boolean' | 'integer' | 'float' | 'date' |
  * @public
  */
 export interface FieldInfo {
+  /** field name */
+  name?: string;
```

Comment:
Since the field info will be used in chart recommendations, getting field name from the field info makes visual mapping easier. And the param 'BasicDataPropertyForAdvice' received by chartAdvisor also has 'name' as property, I think it is better to be consistent on both sides.